mesa/experimental/mesa_signals/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Functionality for Observables."""
+
+from .mesa_signal import All, Computable, Computed, HasObservables, Observable
+from .observable_collections import ObservableList
+
+__all__ = [
+    "Observable",
+    "ObservableList",
```

Comment:
Would an `ObservableDict` (also) be useful, to be able to track which Agent the value originated from?

Reply:
An `ObservableDict` would indeed also be useful. Not sure what you mean with 

> to be able to track which Agent the value originated from

Reply:
I think I see what @EwoutH means here where it would be  something like `{agent.unique_id: <observables>}